src/Shape.java

```diff
@@ -0,0 +1,9 @@
+
+    public interface Shape {
+        double getArea();
+
+        public double calculateAreaOfTheFigure(double a , double h);
```

Comment:
`public` modifier is unnecessary, since all non-statics non-default methods of an interface are public